spyderlib/plugins/__init__.py

```diff
@@ -124,6 +126,14 @@ def __init__(self, *args, **kwargs):
         if sys.platform == 'darwin':
             self.setStyleSheet(self.DARWIN_STYLE)
 
+        # Needed for the installation of the event filter
+        self.title = args[0]
+        self.main = args[1]
+        self.bar = None

```

Comment:
But it's the `tabbar` that holds dockwidgets, right?

So another possibility would be `dock_tabbar` :-)